### PR TITLE
Always show the action column since tags are always enabled

### PR DESCRIPTION
**Pull request by PolariTOON**
#2526 removed the `banks.tags.enabled` flag, effectively enabling tags everywhere.
However, it also introduced some regression in environnements that have the `banks.transaction-actions.deactivated` flag set to `true`, where tags in transaction lists were completely hidden.

This PR makes sure the behavior is the same as it used to be before #2526, when the `banks.tags.enabled` flag was set to `true`.

```
### 🐛 Bug Fixes

* Always show the action column since tags are always enabled
```

### CODE REVIEW

File: src/ducks/categories/CategoriesHeader/MobileFragment.jsx [hunk 1]
```diff
@@ -55,10 +55,6 @@ const MobileFragment = React.memo(props => {
           <HeaderLoadingProgress
             isFetching={!!isFetchingNewData && !isFetching}
           />
-          <LegalMention className="u-flex u-flex-items-center u-flex-justify-around u-mr-1">
-            {incomeToggle}
-          </LegalMention>
-
```
[the remark this answers is not in the document]
Comment (PolariTOON) [reply src/ducks/categories/CategoriesHeader/MobileFragment.jsx:61]: Before #2526, the condition was as follows (take note of the negation):
```
{!flag('banks.tags.enabled') && (
    <LegalMention className="u-flex u-flex-items-center u-flex-justify-around u-mr-1">
        {incomeToggle}
    </LegalMention>
)}
```
Apparently, it was introduced on purpose in cf9c4163b9869db89f64a7ac74378496155aa2d5 .
So if we still want the same behavior, now that tags are always enabled, we should fully remove this part.

**Comment by PolariTOON** [comment]
> So now we always display the column but we don't not display the action inside?

Exactly.
Before tags were a thing, the column was only shown in environnements where `banks.transaction-actions.deactivated` was not set to `true` (i.e. set to `false` or missing). We reused this column to not only put actions but also tags. Now we always need this column for tags. But we still won't show any action if `banks.transaction-actions.deactivated` is set to `true`. That was the behavior prior to #2526, when the `banks.tags.enabled flag` was set to true.
To summarize, now the column is always here, and only a part of its content depends on a flag.

> I really have hard time to understand the fix to be honest.

It might be helpful to read #2526 where the `showTransactionActions || flag('banks.tags.enabled')` condition was turned into `showTransactionActions` instead of `true`.

**Comment by PolariTOON** [comment]
What needs to be done to move this PR forward?

**Review by JF-Cozy — APPROVED** [review]
